addons/web/static/src/core/notebook/notebook.scss

```
@@ -78,4 +77,12 @@
             }
         }
     }
+    @media only screen and (max-width: 767px) {
```

Use[ bootstrap default mixins](https://getbootstrap.com/docs/5.1/layout/breakpoints/#max-width).
Also, missing empty-line before selector
```suggestion

    @media only screen and (max-width: 767px) {
```